templates/repo/issue/milestone_new.tmpl

```diff
@@ -39,23 +38,20 @@
 					<label>{{.locale.Tr "repo.milestones.desc"}}</label>
 					<textarea name="content">{{.content}}</textarea>
 				</div>
-			</div>
-			<div class="ui container">
 				<div class="divider"></div>
-				<div class="ui right">
+				<div>
```

Comment:
I think it needs "gt-text-right", and make the button order as "Cancel" "Submit".

Because, all other Gitea's UI use such layout, it's better to keep consistency.

Reply:
<img width="1390" alt="image" src="https://github.com/go-gitea/gitea/assets/80308335/38f49e95-7ebf-42bd-a2a6-d7073e26a507">